.github/workflows/stale.yml

```diff
@@ -0,0 +1,60 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '0 * * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+        # STALE
+        # Duration of inactivity before making PRs stale
+        days-before-pr-stale: 15
+
+        # Stale Comment
+        stale-pr-message: >
+            This Pull request has been automatically marked as STALE due to inactivity for 15 days
+            and will be CLOSED on further inactivity on the PR for another 15 days.
+
+        # stale label
+        stale-pr-label: 'stale-warning-on-inactivity'
+
+        # remove stale label when updated.
+        remove-stale-when-updated: true
+
+        # CLOSE
+        # Duration of inactivity before closing stale PRs
+        days-before-pr-close: 15
+
+        # closing Comment on the staled PRs
+        close-pr-message: >
+            This pull request has been automatically CLOSED,
+            because there has been no activity for 15 days after marking PR as STALE.
+
+        # close pr label
+        close-pr-label: "closed-due-to-inactivity"
+
+        # Change the order used to fetch the issues and pull requests
+        # So we now start with the oldest PRs and work our way backwards
+        ascending: true
+
+        # labels to expemt from stale action
```

Comment:
labels to expemt from stale action - should be exempt.

Reply:
`labels` can be customized and using default options from the Action, https://github.com/actions/stale#exempt-pr-labels.

Reply:
# labels to expemt from stale action - I was mentioning about the typo correction for 'expemt'.

Reply:
Done. updated.